libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -2,6 +2,8 @@
 // SPDX-License-Identifier: MIT
 
 #include "ebpf_platform.h"
+
+#include "ebpf_low_memory_test.h"
```

Comment:
nit: fix order of listed includes (also note that line 19 is out of alphabetical order, though it was that way before)